File: Iniciante/1041.js
let line1 = lines.shift().split(' ');

let x = Number(line1.shift());
let y = Number(line1.shift());

if (x>0 && y>0){
  console.log("Q1");
} else if (x<0 && y>0){
  console.log("Q2");
} else if (x<0 && y<0){
  console.log("Q3");
} else if (x>0 && y<0){
  console.log("Q4");
} else if (x==0 && y>0 || y<0){
  console.log("Eixo Y");
} else if (y==0 && x>0 || x<0){
  console.log("Eixo X");
} else if (x==0 && y==0){
  console.log("Origem");
}

File: Iniciante/1041.py
linha = input().split()

x = float(linha[0])
y = float(linha[1])

if x > 0 and y > 0:
    print("Q1")
elif x < 0 and y > 0:
    print("Q2")
elif x < 0 and y < 0:
    print("Q3")
elif x > 0 and y < 0:
    print("Q4")
elif x == 0 and y > 0 or y < 0:
    print("Eixo Y")
elif y == 0 and x > 0 or x < 0:
    print("Eixo X")
elif x == 0 and y == 0:
    print("Origem")
